src/events/types.h

```diff
@@ -72,6 +72,13 @@ enum ITCEvent {
 	kITCEventMAX                     ///< For range checks.
 };
 
+/** Enum for keyboard keys. */
+enum Key {
+	kKeyBackspace   = SDLK_BACKSPACE
```

Comment:
Right now, the rest of the code uses the SDLK_* values as is. That should be changed, yes, but that's a job for a different PR. As such, I don't think that enum belongs here yet.